en/04_Changelogs/5.0.0.md

```diff
@@ -30,98 +30,1020 @@ title: 5.0.0 (unreleased)
 
 This is a major release and as a result there are a number of breaking API changes. For a full list of these see [upgrading your project](/upgrading/upgrading_your_project). Some specific details about a few of them are below.
 
-### Removed API (by module, alphabetically) {#api-removed}
+### Removed and changed API (by module, alphabetically) {#api-removed-and-changed}
 
 #### dnadesign/silverstripe-elemental
 
-- Removed deprecated class `DNADesign\Elemental\Search\ElementalSolrIndex`.
-- Removed deprecated class `DNADesign\Elemental\TopPage\TestState`.
-- Removed deprecated method `DNADesign\Elemental\TopPage::getTopPageUpdate()`.
-- Removed deprecated method `DNADesign\Elemental\TopPage::enableTopPageUpdate()`.
-- Removed deprecated method `DNADesign\Elemental\TopPage::disableTopPageUpdate()`.
-- Removed deprecated method `DNADesign\Elemental\TopPage::withTopPageUpdate()`.
+- Removed deprecated method `DNADesign\Elemental\Extensions\ElementalAreaUsedOnTableExtension::updateUsage()`}
+- Removed deprecated method `DNADesign\Elemental\TopPage\DataExtension::disableTopPageUpdate()`}
+- Removed deprecated method `DNADesign\Elemental\TopPage\DataExtension::enableTopPageUpdate()`}
+- Removed deprecated method `DNADesign\Elemental\TopPage\DataExtension::getTopPageUpdate()`}
+- Removed deprecated method `DNADesign\Elemental\TopPage\DataExtension::withTopPageUpdate()`}
+- Changed parameter type in [DNADesign\Elemental\TopPage\SiteTreeExtension::onBeforeDuplicate()](api:DNADesign\Elemental\TopPage\SiteTreeExtension::onBeforeDuplicate()) for `$original` from `Page` to `SilverStripe\CMS\Model\SiteTree`
```

Comment:
```suggestion
- Changed parameter type in [`DNADesign\Elemental\TopPage\SiteTreeExtension::onBeforeDuplicate()`](api:DNADesign\Elemental\TopPage\SiteTreeExtension::onBeforeDuplicate()) for `$original` from `Page` to [`SilverStripe\CMS\Model\SiteTree`](api:SilverStripe\CMS\Model\SiteTree)
```
Use backticks inside the link, since this is still a code reference.
Also link to classes when they're referenced as a type.

Reply:
Note: Only do the type thing if linking for union types also works - e.g:
```md
`[SilverStripe\CMS\Model\SiteTree](api:SilverStripe\CMS\Model\SiteTree)|null`
```